### PR TITLE
Observability TS - TC migration: observability-container-logging

### PR DESCRIPTION
**Pull request by greyerof**

### CODE REVIEW

File: cnf-certification-test/observability/suite.go [hunk 1]
```diff
@@ -17,14 +17,79 @@
...
+	}
+
+	ocpClient := clientsholder.NewClientsHolder()
+
+	// Iterate through all the CUTs to get their log output. The TC checks that at least
+	// one log line is found.
+	numLogLines := int64(1)
+	badContainers := []string{}
+	for _, cut := range env.Containers {
+		podLogOptions := v1.PodLogOptions{TailLines: &numLogLines, Container: cut.Data.Name}
```
Comment (hamadise) [inline cnf-certification-test/observability/suite.go:64]: can you export all this code between lines [64:89] into a separate function ?

Comment (greyerof) [reply cnf-certification-test/observability/suite.go:64]: Ok, will do.

**Review by sebrandon1 — APPROVED** [review]
Looks good pending the suggestion from @hamadise 👍

File: cnf-certification-test/observability/suite.go [hunk 2]
```diff
@@ -17,14 +17,88 @@
...
+func testContainersLogging(env *provider.TestEnvironment) {
+	if len(env.Containers) == 0 {
+		ginkgo.Skip("No containers to run test, skipping")
+	}
+
+	// Iterate through all the CUTs to get their log output. The TC checks that at least
+	// one log line is found.
+	badContainers := []string{}
+	for _, cut := range env.Containers {
+		ginkgo.By(fmt.Sprintf("Checking container %s (pod %s, ns %s) has some logging output", cut.Data.Name, cut.Podname, cut.Namespace))
```
Comment (edcdavid) [inline cnf-certification-test/observability/suite.go:85]: optionally, use the StringShort function to display cut (from merge)

Comment (greyerof) [reply cnf-certification-test/observability/suite.go]: Done.

**Review by edcdavid — APPROVED** [review]

**Pull request merged**